apps/portals/src/portal-components/genie/GenieHomePageHeader.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Box, SxProps, Typography, useMediaQuery } from '@mui/material'
+import Background from './assets/genie-homepage-background.png'
+import GreenPolygon from './assets/Genie-green-rectangle.svg'
```

Comment:
Heads up, since #808 was merged these will have to change

```suggestion
import Background from './assets/genie-homepage-background.png?url'
import GreenPolygon from './assets/Genie-green-rectangle.svg?url'
```